code/datums/datacore.dm

```diff
@@ -254,7 +254,12 @@
 		G.fields["fingerprint"]	= rustg_hash_string(RUSTG_HASH_MD5, H.dna.uni_identity)
 		G.fields["p_stat"]		= "Active"
 		G.fields["m_stat"]		= "Stable"
-		G.fields["sex"]			= H.gender
+		G.fields["sex"] 		= H.dna.features["body_model"] //NSV13 - For Now
```

Comment:
```suggestion
		G.fields["sex"] 		= H.dna.features["body_model"] //NSV13
```
You can probably remove these "for now" parts since Beestation is definitively not having the fields, this applies to all of the instances of NSV tags, not just this one